### PR TITLE
add cluster provider comments

### PR DESCRIPTION
**Pull request by lianghao208**
add comments for clusterprovider.

### CODE REVIEW

File: pkg/clusterprovider/providers/kstone/kstone.go [hunk 1]
```diff
@@ -46,11 +46,13 @@ const (
 	AnnoImportedURI = "importedAddr"
 )
 
+// EtcdClusterKstone is the etcd cluster created from kstone-dashboard
```
Comment (tangcong) [inline pkg/clusterprovider/providers/kstone/kstone.go:49]: // EtcdClusterKstone is responsible for synchronizing kstone.tkestack.io/etcdcluster to kstone-etcd-operator which will be responsible for the real implementation of the etcd cluster.

**Review by tangcong — CHANGES_REQUESTED** [review]
thanks, a little suggestion，please refer to this commit specification.
image

File: pkg/clusterprovider/providers/kstone/kstone.go (hunk 1 above)
Comment (lianghao208) [reply pkg/clusterprovider/providers/kstone/kstone.go]: done

**Comment by lianghao208** [comment]
@tangcong 
Thanks for the suggestion, I've update the commit message, PTAL.

**Review by tangcong — APPROVED** [review]

**Pull request merged**